.github/workflows/docs.yml

```
@@ -0,0 +1,27 @@
+        with:
+          # Check out the merge commit on the base branch
+          ref: ${{ github.event.pull_request.merge_commit_sha }}
+      - name: Your step
+        run: |
+          curl -L  \
+            -X POST \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer ${{ secrets.DOCS_REPO_TOKEN }}" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            https://api.github.com/repos/ozontech/seq-db-docs/dispatches \
+            -d '{"event_type":"dispatch-event"}'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)